### DISCUSSION TITLE
Alarm/Stopping after first cycle

### DISCUSSION THREAD

**Discussion by meinbeitrag** [arkypita/LaserGRBL#916] [Q&A]
Hello and first of all: thank you for the great software!

I am completely new and have the problem, that the laser or software is stopping after the first cycle. When I want to cut wood I have always to make one cycle and repeat it manually. I use an Ortur Laser Master 15W. Everything is fine but this happens all the time. I use vectors for this cutting.

Any suggestions? I am not a programmer. So easy language would be appreciated. :) 

Best wishes,
Matthias

**Comment by meinbeitrag** [comment]
Additional information: I use LaserGRBL v3.1.2
I updated the Ortur with the new firmware.

The cause is: StopResponding

**Comment by StuartB4** [comment]
Even if you set the number of passes?
![tytyt](https://user-images.githubusercontent.com/16980026/76691396-49c8ea00-6642-11ea-8d54-4b9035a27807.jpg)

**Comment by meinbeitrag** [comment]
It is the only way I know for cycles (passes).
Is there another way?

**Comment by StuartB4** [comment]
Not as far as I know. Have you tried changing this setting to a slower one.

![yuyuyu](https://user-images.githubusercontent.com/16980026/76691936-0cfff180-6648-11ea-8dd1-7aaa3eb4ca16.jpg)

**Comment by meinbeitrag** [comment]
Hello,

yes, I already reduced it to "Fast". Maybe I have to reduce it even more.
But the strange thing is, that it mostly happens when one cycle is finished.
I also tried all Streaming Mode settings. Did not help.

**Comment by meinbeitrag** [comment]
Even with "Slow" it does not work. Here is the message I get. 
On the left you can see in blue: "MSG:Reset to continue". I guess this is important but I don't know what to do.
![image](https://user-images.githubusercontent.com/38387222/76700968-0f506300-66bd-11ea-87fe-3dd7cdc02b06.png)

**Comment by StuartB4** [comment]
You should be able to click the Lightning icon next to the padlock to reset.
Do you have anything in the settings that are adding extra gcode commands to your file.
If you do make a note of them so you know what they are, then remove them and try again.
You can always replace them if you need to.

![ljgfhg](https://user-images.githubusercontent.com/16980026/76701549-1a55c400-66ba-11ea-828c-0e5a33438c50.jpg)

**Comment by arkypita** [comment]
> The cause is: StopResponding
> 
> yes, I already reduced it to "Fast". Maybe I have to reduce it even more.
> But the strange thing is, that it mostly happens when one cycle is finished.
> I also tried all Streaming Mode settings. Did not help.

Yes, it is very strange.

Stop responding is signaled by LaserGRBL when grbl (the firmware that is running on controller board) does not reply to status query for more than an amount of seconds.

What version of grbl do you have on your board?

**Comment by meinbeitrag** [comment]
Hm, I don`t know. How can I find out? Haven't found any "information"-button for this. I only see the "Flash GRBL Firmware" under "Tools".

**Comment by StuartB4** [comment]
Should tell you here when you are connected.

![7](https://user-images.githubusercontent.com/16980026/76704815-724cf480-66d3-11ea-9885-a9d0b725630e.jpg)

**Comment by meinbeitrag** [comment]
Hello, 

I also have Grbl 1.1f. Shall I change this?
But I am not sure which Arduino board I have. 
I am a little bit Arduino experienced. But on the machine, it does not loke like any Arduino I know. 
It is small and long so maybe a "nano". But not sure. 

![image](https://user-images.githubusercontent.com/38387222/76746363-ada0ff00-6777-11ea-8de6-fd23ae4e0625.png)

**Comment by StuartB4** [comment]
No need to change grbl, 1.1f is fine. Yes it may be a Nano if it's plugged in to the main control board.
It may be Ortur's own board. Do you have a picture?

Also, can you upload a screenshot of the GRBL settings.
Just start LaserGRBL and click Connect.
Goto Grbl menu and click Grbl configuration.
When the list pops up just stretch it a bit to show all the $ settings then get a screenshot
and post it here.

**Comment by meinbeitrag** [comment]
Hey,

thank you for your great help! It is nearly impossible to take a picture of the board without disassembling the Ortur Laser Master. Is it necessary if Grbl 1.1f is fine?

I have made the screenshot. :) 

![image](https://user-images.githubusercontent.com/38387222/76775149-3aae7d00-67a5-11ea-9f8c-cb027cb5003e.png)

**Comment by meinbeitrag** [comment]
I think I have solved it. I found the menu "multiple passes". I have seen that there is a Z axis.
I deleted everything and wrote "G90" in this. Now it makes multiple passes.

![image](https://user-images.githubusercontent.com/38387222/76776173-b1984580-67a6-11ea-9413-c2b00acbd9be.png)
